include/cpp2regex.h

```diff
@@ -1555,6 +1556,26 @@ if (((*this) & perl_code_syntax_in_classes) == perl_code_syntax_in_classes) {_re
 return cpp2::move(_ret) + ")"; 
 }
 
+[[nodiscard]] auto expression_flags::from_string(cpp2::impl::in<std::string_view> s) -> expression_flags{
```

Comment:
@hsutter Why do all of these lines start at column 0?  There is leading whitespace in the metafunction.  Is it because the whitespace is tabs, and those are being stripped somehow?

Reply:
Good question, I don't know.